rcl_action/test/rcl_action/test_goal_handle.cpp

```diff
@@ -161,7 +161,7 @@ using EventStateActiveCancelableTuple =
   std::tuple<rcl_action_goal_event_t, rcl_action_goal_state_t, bool, bool>;
 using StateTransitionSequence = std::vector<EventStateActiveCancelableTuple>;
 const std::vector<std::string> event_strs = {
-  "EXECUTE", "CANCEL", "SET_SUCCEEDED", "SET_ABORTED", "SET_CANCELED"};
+  "EXECUTE", "CANCEL", "SUCCEED", "ABORT", "CANCEL"};
```

Comment:
@jacobperron shouldn't it be:
```suggestion
  "EXECUTE", "REQUEST_CANCEL", "SUCCEED", "ABORT", "CANCEL"};
```
instead?

Reply:
Yep, good catch. fixed in 225d805